src/main/java/com/saralein/server/validation/ArgsValidation.java

```diff
@@ -0,0 +1,93 @@
+package com.saralein.server.validation;
+
+import static com.saralein.server.Constants.*;
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+public class ArgsValidation {
```

Comment:
This class seems to be doing more than just argument validation. It is also parsing....

We might not need to do it now, but I would recommend you think about how this class can evolve to be more Open/Closed.